test/threads.jl

```diff
@@ -95,10 +95,21 @@ end
 # hang in FreeBSD. So, it's tested only in Linux and Windows:
 const AFFINITY_SUPPORTED = (Sys.islinux() || Sys.iswindows()) && !running_under_rr()
 
+# If we're running inside of a cpuset, we need to test affinity with
+# valid cpu core choices
+function get_cpumask()
+    self = ccall(:uv_thread_self, Ptr{Cvoid}, ())
+    masksize = ccall(:uv_cpumask_size, Cint, ())
+    mask = zeros(UInt8, masksize)
+    ccall(:uv_thread_getaffinity, Cint, (Ptr{Ptr{Cvoid}}, Ptr{UInt8}, Cint), Ref(self), mask, masksize)
+    return mask
+end
```

Comment:
We already have a similar function in `print_process_affinity.jl`. So, I think we just need to import it here (or maybe somewhere at the beginning of the file).

```suggestion
include("print_process_affinity.jl")
```